Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN sh -c 'echo "deb http://packages.ros.org/ros/ubuntu $(lsb_release -sc) main"
     && rosdep init
 
 RUN apt-get update && \
-        apt-get install -y \
+        DEBIAN_FRONTEND=noninteractive apt-get install -y \
```

Comment:
Were there issues without this? I don't see any harm in adding it, but I'm curious.

Reply:
I haven't tried it without it because it was used in the docker-ros-box code I was referencing. All it does is choose the default answer for all prompts.